idaes/models/unit_models/tests/test_heat_exchanger_1D.py

```diff
@@ -868,49 +868,44 @@ def test_get_stream_table_contents(self, iapws):
-                "Mass Flow (kg/s)": pytest.approx(0.090076, rel=1e-4),
-                "T (K)": pytest.approx(422.6, rel=1e-4),
-                "P (Pa)": pytest.approx(101325, rel=1e-4),
+                "Molar Flow": pytest.approx(5, rel=1e-4),
+                "Mass Flow": pytest.approx(0.090076, rel=1e-4),
+                "T": pytest.approx(422.6, rel=1e-4),
+                "P": pytest.approx(101325, rel=1e-4),
                 "Vapor Fraction": pytest.approx(1, abs=1e-4),
-                "Molar Enthalpy (J/mol) Vap": pytest.approx(50000, rel=1e-4),
-                "Molar Enthalpy (J/mol) Liq": pytest.approx(11342, rel=1e-4),
+                "Molar Enthalpy": pytest.approx(50000, rel=1e-4),
             },
             "Hot Side Outlet": {
-                "Molar Flow (mol/s)": pytest.approx(1, rel=1e-4),
-                "Mass Flow (kg/s)": pytest.approx(1.8015e-2, rel=1e-4),
-                "T (K)": pytest.approx(286.34, rel=1e-4),
-                "P (Pa)": pytest.approx(1e5, rel=1e-4),
+                "Molar Flow": pytest.approx(1, rel=1e-4),
+                "Mass Flow": pytest.approx(1.8015e-2, rel=1e-4),
+                "T": pytest.approx(270.4877112932641, rel=1e-4),
+                "P": pytest.approx(11032305.8275, rel=1e-4),
                 "Vapor Fraction": pytest.approx(0, abs=1e-4),
-                "Molar Enthalpy (J/mol) Vap": pytest.approx(2168.6, rel=1e-4),
-                "Molar Enthalpy (J/mol) Liq": pytest.approx(1000, rel=1e-4),
+                "Molar Enthalpy": pytest.approx(0.01102138712926277, rel=1e-4),
             },
             "Cold Side Inlet": {
-                "Molar Flow (mol/s)": pytest.approx(5, rel=1e-4),
-                "Mass Flow (kg/s)": pytest.approx(0.090076, rel=1e-4),
-                "T (K)": pytest.approx(365.88, rel=1e-4),
-                "P (Pa)": pytest.approx(101325, rel=1e-4),
+                "Molar Flow": pytest.approx(5, rel=1e-4),
+                "Mass Flow": pytest.approx(0.090076, rel=1e-4),
+                "T": pytest.approx(365.88, rel=1e-4),
+                "P": pytest.approx(101325, rel=1e-4),
                 "Vapor Fraction": pytest.approx(0, abs=1e-4),
-                "Molar Enthalpy (J/mol) Vap": pytest.approx(47926, rel=1e-4),
-                "Molar Enthalpy (J/mol) Liq": pytest.approx(7000, rel=1e-4),
+                "Molar Enthalpy": pytest.approx(7000.0, rel=1e-4),
             },
             "Cold Side Outlet": {
-                "Molar Flow (mol/s)": pytest.approx(1, rel=1e-4),
-                "Mass Flow (kg/s)": pytest.approx(1.8015e-2, rel=1e-4),
-                "T (K)": pytest.approx(286.34, rel=1e-4),
-                "P (Pa)": pytest.approx(1e5, rel=1e-4),
+                "Molar Flow": pytest.approx(1, rel=1e-4),
+                "Mass Flow": pytest.approx(1.8015e-2, rel=1e-4),
+                "T": pytest.approx(270.4877112932641, rel=1e-4),
+                "P": pytest.approx(11032305.8275, rel=1e-4),
```

Comment:
Are these changes in values expected? I notice that P has increased significantly.

Reply:
They aren't results.  These are pre solve unit tests and the new property package has different default values.  They should be overwritten in initialization, but it hasn't been run for these tests.